src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileInfo.cs

```diff
@@ -29,13 +29,12 @@ public PhysicalFileInfo(FileInfo info)
 
         public Stream CreateReadStream()
         {
-            // Note: Buffer size must be greater than zero, even if the file size is zero.
```

Comment:
I thought I verified 0 length file and it worked, but I re-tested and it didn't. Thank you for noticing.